interfaces/automated-testing/geo-awareness/geo-awareness.yaml

```diff
@@ -41,9 +58,55 @@ components:
           type: string
           example: v0.0.1-445ad3
 
+    InjectGeozonesRequest:
+      type: object
+      properties:
+        source:
```

Comment:
We'll probably add other sources in the future that aren't HTTPS sources (e.g., AMQP), so let's name this field `https_source`

Reply:
I was thinking to use the same field with different types of objects and therefore add another ref at line 67 for AMQP for instance. Isn't it correct as is? Happy to change if you think multiple keys is a better option.

Reply:
OpenAPI allows a field to contain AnyOf multiple types (so the field can contain data which may be one of multiple different types) because it's based on JSON validation, but we generally want to avoid that because lots of code generation tools do not like polymorphic data types.  A more common-denominator approach is to have each field contain only one data type, and then specify just one of the field options.  An example of this is `outline_circle` and `outline_polygon` in `Volume3D` of the UTM API (rather than just `outline` with AnyOf `Circle` or `Polygon`).  Since the data type for this field is (currently) `GeozonesHttpsSource`, I think `https_source` is the most appropriate name for the field.